lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -7,6 +7,13 @@ module ShopifyTransporter
   module Exporters
     module Magento
       class Soap
+        class FailedLoginError < Exporters::ExportError
+          def initialize(error_message)
+            sep = $INPUT_RECORD_SEPARATOR
+            super("Unable to obtain SOAP session ID from server.#{sep + sep}Details:#{sep + sep + error_message}")
```

Comment:
Super Minor, I vote for one less `sep` after `Details:`

Reply:
Done!